frontend/src/components/Links/Settings/Settings.jsx

```
@@ -43,35 +46,33 @@
-      (acc, it) => ({
-        ...acc,
-        [it.name]: it.name === "target" ? it.value === "true" : it.value,
-      }),
-      {}
-    );
+  const handleClose = async (e, info) => {
+    e?.preventDefault();
+    if (!info) {
+      info = Array.from(settingsRef.current.querySelectorAll('input')).reduce(
+        (acc, it) => ({
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)